### DISCUSSION TITLE
Same barcode assigned to two different cellGroups in clones.tsv file from 10X TCR data

### DISCUSSION THREAD

**Discussion by genome2020** [milaboratory/mixcr#1808] [Q&A] [answered]
Hi,

I recently ran MiXCR v.4.7.0 with my 10X TCR data on multiple samples. Here is the exact command I used.

mixcr -Xmx100g analyze 10x-sc-xcr-vdj --threads 8 --species hsa sample1_R1_001.fastq.gz sample1_R2_001.fastq.gz sample1

For some samples I noticed same cell barcode with different 'cellGroup' in the clones.tsv file. See example:

<img width="1630" alt="Screen Shot 2024-09-30 at 3 20 31 PM" src="https://github.com/user-attachments/assets/01490d29-a2e4-454b-bd94-094d9fb5cba5">

I would expect that cellGroup for the same barcode would be the same as I would expect all the TCR chains corresponding to same barcode/cellGroup belonging to the same cell. Could you please explain why there are two cellGroups for the same barcode?

**Comment by mizraelson** [comment, marked as answer]
It’s hard to say without closer investigation, but generally speaking, this can happen in cases when multiple cells are captured in the same droplet. If MiXCR finds enough information in the dataset, it can identify that two different cells are marked with the same barcode ID. For example, imagine you have 100 cells with clones AB and 500 cells with clones CD. If you then find a cell with both AB and CD clones in it, MiXCR will assign them to the according cell groups.

**Reply by genome2020** [reply]
Thanks for the quick reply @mizraelson
